README.md

```diff
@@ -122,145 +162,143 @@ firmwares/
+A build is triggered by any PR or commit to the master branch. We switched from a self-hosted 
+[Buildbot setup](https://buildbot.net/) to GitHub Actions because it reduced the amount of maintenance we had to do.
 
-Since the switch to GitHubActions nobody reimplemented a way of deploying / publishing the builds, so 
-the following sentences are just some kind of reminder for this ToDo.
+Since the switch to GitHub Actions, no one has implemented a way to deploy / publish the builds, so the 
+the following sentences are just a kind of reminder for this to-do.
 
-~~All branches whose name complies to the "X.Y.Z" pattern are built and put into the "stable" downloads directory:
-[http://buildbot.berlin.freifunk.net/buildbot/stable/](http://buildbot.berlin.freifunk.net/buildbot/stable/)~~
+~~All branches whose name matches the pattern "X.Y.Z" are built and placed in the "stable" download directory:
+[https://buildbot.berlin.freifunk.net/buildbot/stable/](https://buildbot.berlin.freifunk.net/buildbot/stable/)~~
 
-~~All branches with names not fitting the "X.Y.Z" pattern are built and put into the "unstable" directory:
-[http://buildbot.berlin.freifunk.net/buildbot/unstable/](http://buildbot.berlin.freifunk.net/buildbot/unstable/)
-Note that in the directory there is no reference to the branch name; unstable builds can be identified by build number only.~~
+~~All branches with names that do not match the pattern "X.Y.Z" are built and placed in the "unstable" directory:
+[https://buildbot.berlin.freifunk.net/buildbot/unstable/](https://buildbot.berlin.freifunk.net/buildbot/unstable/)
+Note that there is no reference to the branch name in the directory; unstable builds can only be identified by the build number.~~
 
 #### Creating a release
 
-Every release has a [semantic version number](http://semver.org); each major version has its own codename.
-We name our releases after important female computer scientists, hackers, etc.
-For inspiration please take a look at the related
+Each release has a [semantic version number] (https://semver.org/); each major release has its own code name.
+We name our releases after important computer scientists, hackers, etc.
+For inspiration, please see the related
 [ticket](https://github.com/freifunk-berlin/firmware/issues/24).
 
-For a new release, create a new branch. The branch name must be a semantic version
-number. Make sure you change the semantic version number and, for major releases,
-the codename in the README and config files (./configs/*)
+A new branch must be created for a new release. The branch name must be a semantic version number.
+Make sure that you include the semantic version number and, for major releases
+the codename in the README and configuration files (./configs/*).
 
-~~The buildbot will build the release and place the files in the stable direcotry
-once you pushed the new branch to github.~~
+~~The buildbot creates the version and puts the files in the stable directory
```

Comment:
I think you should remove any comments of buildbot, since it's used to build the more active [Falter-firmware](https://github.com/Freifunk-Spalter). Hedy (this Firmware) uses github-actions to build the source code.

Reply:
It was intentional to keep them in the document, to just update them later when some publishing is re-established. But the note can always restored from the history.